apps/src/codegen.js

```diff
@@ -451,6 +451,32 @@ exports.isAceBreakpointRow = function (session, userCodeRow) {
   return Boolean(bps[userCodeRow]);
 };
 
+/**
+ * Selects code in droplet/ace editor.
+ *
+ * This function simply highlights one spot, not a range. It is typically used
+ * to highlight where an error has occurred.
+ */
+exports.selectEditorRowCol = function (editor, row, col) {
+  if (editor.currentlyUsingBlocks) {
+    var style = {color: '#FFFF22'};
```

Comment:
Unfortunately, droplet's API takes a parameter called `style` which is a object that contains a property called color. But not a real CSS style or class. We could move to at least create a constant in our code for this color, though.